config.json

```diff
@@ -29,5 +29,9 @@
             "email": "tbembersimeao@ufl.edu",
             "institution": "University of Florida - CTSI"
         }
-    ]
+    ],
+    "enable-every-page-hooks-on-system-pages": true,
+    "compatibility": {
+        "redcap-version-min": "8.0.0"
```

Comment:
8.0.3 is required for semantic versioning.

Reply:
Done!